migrations/initdb-banners.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable new-cap */
+
+'use strict';
+
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}banners`;
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.sequelize.transaction(async (transaction) => {
+            await queryInterface.createTable(table, {
+                id: {
+                    allowNull: false,
+                    autoIncrement: true,
+                    primaryKey: true,
+                    type: Sequelize.INTEGER
+                },
+                message: {
+                    type: Sequelize.STRING(512)
+                },
+                isActive: {
+                    type: Sequelize.BOOLEAN
+                },
+                createTime: {
+                    type: Sequelize.STRING(32)
+                },
+                createdBy: {
+                    type: Sequelize.STRING(128)
+                },
+                type: {
+                    type: Sequelize.STRING(32)
+                }
+            }, { transaction }
+            );
+
+            await queryInterface.addConstraint(table, ['message', 'createTime', 'type'],
+                {
+                    name: `${table}_message_createTime_type_key`,
+                    type: 'unique',
+                    transaction
+                }
+            );
+        });
+    },
+    // eslint-disable-next-line no-unused-vars
+    down: (queryInterface, Sequelize) => queryInterface.dropTable(table)
+};
```

Comment:
I know this is u likely but is it ok to not have droptable for “down”? I get that this is first iteration from the existing FB schemas, just hope to reduce the dummy mistake interface

Reply:
Yes, will remove that. It can be controlled via DB privileges as well. Ideally, dropTable will be a rare scenario.